app/views/users/new.html.erb

```diff
@@ -0,0 +1,56 @@
+<h1>User#new Form</h1>
+<!--- <form method='POST' action='/users' accept-charset="UTF-8"> 
+<input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %> >
+<label for='usename'>Username: </label>
+<input type='text' name='user[username]' placeholder='Enter your username'><br>
+<label for='email'>Email: </label>
+<input type='email' name='user[email]' placeholder='Enter your email'><br>
+<label for='password'>Password: </label>
+<input type='password' name='user[password]' placeholder='Enter password at least 8 alphanum'><br>
+<input value='submit' type='submit'>
+</form> --->
+
+
+ <!--- <%= form_tag('/users', method: 'post') do %>
+	<p>
+		 <%= label_tag(:username, "Username") %> 
+  	<%= text_field_tag(:username) %>
+	</p> 
+
+   <p> 
+	 	 <%= label_tag(:email, "Email") %>
+		<%= email_field_tag(:email) %> 
+	</p>
+
+	<p>
+		<%= label_tag(:password, "Password") %>
+  	<%= password_field_tag(:password) %>
+	</p>
+	  <%= submit_tag("Submit") %>
+<% end %> --->
+
+<!--- <%= form_with model: @user do |form| %>
+  <%= form.text_field :username, placeholder: "Username" %><br />
+
+  <%= form.email_field :email, placeholder: "Email@example.com" %><br />
+
+  <%= form.password_field :password, placeholder: "Password" %><br />
+
+	<%= form.submit %>
+<% end %> --->
+
+<%= form_for(@user) do |form|%>
```

Comment:
Remove the `form_for` tag and render your `form` partial here instead. Hint:
`<%= render 'form'%>`